talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,254 @@
+            }
+            //updating displayName
+            currentBuild.displayName = "#" + currentBuild.number + ": " + branchName + "(" + runtimeName + ")"
+          }
+        }
+      }
+    }
+    stage('Test Checkout') {
+      steps {
+        container(tsbiImage) {
+          // Checkout of the connector directory (TODO Will be cleaned when test component will be ready)
+          checkout([
+              $class           : 'GitSCM',
+              branches         : [[name: '*/master']],
+              extensions       : [
+                  [$class           : 'RelativeTargetDirectory',
+                   relativeTargetDir: './connectors'],
+                  [$class                           : 'CleanBeforeCheckout',
+                   deleteUntrackedNestedRepositories: true]],
+              userRemoteConfigs: [[credentialsId: 'github-ssh',
+                                   url          : 'git@github.com:Talend/connectors-se.git']]
+          ])
+          // Checkout of the test directory from API Tester
+          checkout([
+              $class           : 'GitSCM',
+              branches         : [[name: "*/${branchToUse}"]],
+              extensions       : [
+                  [$class           : 'RelativeTargetDirectory',
+                   relativeTargetDir: './test_scenarios'],
+                  [$class                           : 'CleanBeforeCheckout',
+                   deleteUntrackedNestedRepositories: true],
+                  [$class             : 'SparseCheckoutPaths',
+                   sparseCheckoutPaths: [[path: 'talend-component-maven-plugin/src/it/web/test']]]
+              ],
+              userRemoteConfigs: [[credentialsId: 'github-ssh',
+                                   url          : 'git@github.com:Talend/component-runtime.git']]
+          ])
+        }
+      }
+    }
+
+    stage('Connector Build') {
+      steps {
+        container(tsbiImage) {
+          script {
+            if (runtimeVersion != "default"){
+                sh """
+                    chmod +x ${SCRIPTS_PATH}/pom-edit.sh
+                    ${SCRIPTS_PATH}/pom-edit.sh ${runtimeVersion} ${CONNECTORS_PATH}
+                    """
+            }
+            sh """
+                mvn -f ${TESTED_CONNECTOR_PATH}/pom.xml clean install -DskipTests -Dspotless.apply.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true
+                """
+          }
+        }
+      }
+    }
+
+    stage('TCK component server start') {
+      steps {
+        container(tsbiImage) {
+          sh """
+              chmod +x ${SCRIPTS_PATH}/server-start.sh
+              ${SCRIPTS_PATH}/server-start.sh ${TESTED_CONNECTOR_PATH} ${serverLog} ${serverPort} ${serverTimeout} &
+              """
+          sh """
+              chmod +x ${SCRIPTS_PATH}/server-check.sh
+              ${SCRIPTS_PATH}/server-check.sh ${serverPort}
+              """
+          sh """
+              cat ${TESTED_CONNECTOR_PATH}/${serverLog}
```

Comment:
made another comment on log file, forget my other comment.
But, you don't need to redirect to a log file, leave standard output and you'll have the same result.

Reply:
Moreover, this is a bit ugly ;-)

Reply:
It is to save it as an artifact in jenkins

Reply:
This part has been merged to be nicer